docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -10,6 +10,15 @@ Compatibility
 
 The Pinot connector is compatible with all Pinot versions starting from 0.1.0.
 
+Requirements
+------------
+
+To connect to Pinot, you need:
+
+* Pinot 0.1.0 or higher.
+* Network access from the Trino coordinator and workers to the Pinot Server.
+  Port 8098 is the default port.
```

Comment:
Really .. not 9000 like in the examples ?

Reply:
I changed the port to 9000.